configs/14.0/packages/glibc/stage_optimized

```diff
@@ -84,7 +84,8 @@ atcfg_configure() {
 	CXX="${at_dest}/bin/g++ -m${AT_BIT_SIZE}" \
 	CFLAGS="-g -O3 \
 		${with_longdouble:+-mlong-double-128} \
-		-fcommon" \
+		-fcommon \
+		-Wno-error=maybe-uninitialized" \
```

Comment:
Is there an issue open to keep track of this?

Reply:
Done with issue #1635 .